src/Writers/HTMLWriter.jl

```diff
@@ -958,9 +1038,25 @@ function render_head(ctx, navnode)
         script[:src => relhref(src, ctx.themeswap_js)],
         # Custom user-provided assets.
         asset_links(src, ctx.settings.assets),
+        asset_links(src, ctx.asset_list),
     )
 end
 
+"""
+    plugin_html_assets(::Documenter.Plugin)::Vector{HTMLAsset}
+
+This function allows plugins to register additional assets to the `HTMLWriter`.
+It should return a vector of [`HTMLAsset`](@ref) simillar to the example of
+[`asset`](@ref).
+
+!!! info "Note:"
+    Please note that contrary to the regular use of the [`asset`](@ref)
+    function `src` has to be provided as absolute path for local assets!
+"""
+function plugin_html_assets(::Documenter.Plugin)::Vector{HTMLAsset}
+    return HTMLAsset[] # return vector of assets which will be added
+end
```

Comment:
We can simplify this down to a one-liner:

```suggestion
plugin_html_assets(::Documenter.Plugin) = HTMLAsset[]
```

Regarding the `::Vector{HTMLAsset}` return type part -- as you said, this only applies to this particular method, so it's superfluous here (as the function body is trivial). However, we do want to keep it in the docstring, since we expect the plugin writers to return `::Vector{HTMLAsset}` (although, in principle, anything iterable with `eltype()=HTMLAsset` would also be okay).

Reply:
I agree and changed it that way.